src/os/bluestore/BlueStore.cc

```diff
@@ -4107,6 +4107,8 @@ void BlueStore::_init_logger()
 		    "collection");
   b.add_u64_counter(l_bluestore_read_eio, "bluestore_read_eio",
                     "Read EIO errors propagated to high level callers");
+  b.add_u64(l_bluestore_fragmentation, "bluestore_fragmentation_percentage",
+            "How fragmented bluestore free space is");
```

Comment:
Can you include teh definition here?  (free extents / max possible number of free extents)

Reply:
will do